gammapy/utils/tests/test_time.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```

Comment:
We can remove it when we drop Python 2 support. I.e. when Sherpa supports Python 3 and > 90% of our users are on Python 3, i.e. maybe in a few years, maybe never. :-)